ci-operator/config/openshift/knative-client/openshift-knative-client-release-v0.1.0.yaml

```diff
@@ -0,0 +1,29 @@
+tag_specification:
+  name: '4.1'
+  namespace: ocp
+promotion:
+  cluster: https://api.ci.openshift.org
+  namespace: openshift
+  name: release-v0.1.0
+base_images:
+  base:
+    name: '4.0'
```

Comment:
Your `tag_specification` here is 4.1 but your base is 4.0?

Reply:
updated

Reply:
thanks!